content/en/community/code/continuous-integrating-jx-itself.md

```diff
@@ -34,7 +34,7 @@ One of the most common e2e test is to create a new application, push it to a new
 
 The last command on the bash script is actually running the tests using the jx step bdd command, that basically clones [the tests repository](https://github.com/jenkins-x/bdd-jx) and runs a [Makefile target in that repository](https://github.com/jenkins-x/bdd-jx/blob/master/Makefile). Some of the logic executed by Jenkins X during the e2e tests is executed using the jx binary that we compiled on the first step of the pipeline. But not all. Other parts of the logic is executed inside the containers used on the steps of the pipeline.
 
-The containers used as part of the Jenkins X pipelines are called builders. If we want our pull request changes to be used in the builder containers too, we need to tell Jenkins X to use the container images published on a previous step of the pipeline, which contain the pull request changes. Fortunately, [Jenkins X allows you to pass a file that will override the "values.yaml" file used to install Helm charts as part of the Jenkins X installation](/docs/guides/managing-jx/common-tasks/config/). If you don't use a custom values file, [this is the default used when installing Jenkins X](https://github.com/jenkins-x/jenkins-x-platform/blob/master/jenkins-x-platform/values.yaml).
+The containers used as part of the Jenkins X pipelines are called builders. If we want our pull request changes to be used in the builder containers too, we need to tell Jenkins X to use the container images published on a previous step of the pipeline, which contain the pull request changes. Fortunately, [Jenkins X allows you to pass a file that will override the "values.yaml" file used to install Helm charts as part of the Jenkins X installation](/docs/resources/guides/managing-jx/common-tasks/config/). If you don't use a custom values file, [this is the default used when installing Jenkins X](https://github.com/jenkins-x/jenkins-x-platform/blob/master/jenkins-x-platform/values.yaml).
```

Comment:
I think it would be nice to move to a "one sentence per line" approach eventually. That makes digging content much easier. @jha-cloudbees - has this been discussed before?

Reply:
It hasn't come up yet in the CloudBees Docs team style discussions, but it could be something to discuss in the next Jenkins X docs SIG meeting!

Reply:
> It hasn't come up yet in the CloudBees Docs team style discussions, but it could be something to discuss in the next Jenkins X docs SIG meeting!

+100, I think it makes a lot of sense. Happy to make the case for it. I for my part already do this for all the pages I edited. It makes things easier on multiple levels.